projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -170,7 +221,9 @@ export const mapDispatchToProps = ( dispatch, featureSlug ) => {
 		case 'videopress':
 			return {
 				activateFeature: () => {
-					return dispatch( updateSettings( { videopress: true } ) );
+					return restApi.installPlugin( 'videopress', 'recommendations' ).then( () => {
```

Comment:
Was this working? Or are we circumventing that by getting rid of the CTA if videopress is not installed [here](https://github.com/Automattic/jetpack/pull/26484/files#diff-27a0ea9326bc6bff96c64d327b5f9bb7b25b819511b6e35033cc4522f2ff7ac8R149)

Reply:
It stopped working after the introduction of a standalone JP VideoPress plugin. So I've updated it to the recent situation.

Reply:
Ok that is what I thought, thanks!